libvast/vast/system/partition_client_actor.hpp

```diff
@@ -0,0 +1,29 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "vast/fwd.hpp"
+
+#include <caf/typed_event_based_actor.hpp>
+
+namespace vast::system {
+
+/// The PARTITION CLIENT actor interface.
+using partition_client_actor = caf::typed_actor<
+	// The client receives several sets of ids followed by a final
+	// `atom::done`, which as sent as response message to the
```

Comment:
Why is the `done` that is mentioned here not listed below?

Reply:
It's not a free message but sent as replay to the initial request. I've reworded the comment to make it more clear.

Reply:
I think in the context of a partition_client_actor that is irrelevant.